src/codemagic/tools/_app_store_connect/action_groups/builds_action_group.py

```diff
@@ -68,6 +71,58 @@ def get_build(self, build_id: ResourceId, should_print: bool = True) -> Build:
 
         return self._get_resource(build_id, self.api_client.builds, should_print)
 
+    @cli.action(
+        'expire',
+        BuildArgument.BUILD_ID_RESOURCE_ID,
+        action_group=AppStoreConnectActionGroup.BUILDS,
+    )
+    def expire_build(
+        self,
+        build_id: ResourceId,
+        should_print: bool = True,
+    ) -> Build:
+        """
+        Expire a specific build, an expired build becomes unavailable for testing
+        """
+
+        return self._modify_resource(
+            self.api_client.builds,
+            build_id,
+            should_print,
+            expired=True,
+        )
+
+    @cli.action(
+        'expire-builds',
+        AppArgument.APPLICATION_ID_RESOURCE_ID,
+        BuildArgument.BUILD_ID_RESOURCE_ID_EXCLUDE_OPTIONAL,
+        action_group=AppStoreConnectActionGroup.BUILDS,
+    )
+    def expire_builds(
```

Comment:
I feel like this method would be better suited in apps action group with method name `expire_app_builds`. Action name could remain the same, but it would be in a different namespace that reflects more appropriately the meaning.

Reply:
Remark about method name still stands. Action name is fine as is.